### PR TITLE
Postfix log parser

### PR DESCRIPTION
**Pull request by sathieu**

### CODE REVIEW

**Comment by sathieu** [comment]
This is WIP, please do not merge!

**Comment by sathieu** [comment]
Hello,

I need some help to fix the testsuite. Thanks

**Comment by sathieu** [comment]
Can someone review. I think it's OK now.

File: modules/postfix.lua [hunk 1]
```diff
@@ -0,0 +1,500 @@
+-- This Source Code Form is subject to the terms of the Mozilla Public
+-- License, v. 2.0. If a copy of the MPL was not distributed with this
+-- file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+local ip = require 'ip_address'
+local l = require 'lpeg'
+l.locale(l)
+
+local M = {}
+-- setfenv(1, M) -- Remove external access to contain everything in the module
```
Comment (rafrombrc) [inline modules/postfix.lua]: This shouldn't be commented out.

Comment (sathieu) [inline modules/postfix.lua]: But it breaks rawset!

Comment (rafrombrc) [inline modules/postfix.lua]: You have to add anything from the global namespace to your local namespace before the `setfenv` call. So adding `local rawset = rawset` around line 8 should do the trick. See similar usage in the [syslog module](https://github.com/mozilla-services/lua_sandbox/blob/dev/modules/syslog.lua#L12).

**Comment by sathieu** [comment]
Replaced by #95

**Pull request closed without merging**